### ISSUE TITLE
[Feature] warning on trace viewer when opening a trace file generated with a different version of playwright

### ISSUE THREAD

**Issue by DetachHead** [microsoft/playwright#21898]
the 1.32.0 trace viewer seems to be incompatible with trace files generated with older versions of playwright. specifically, the DOM viewer seems to be broken and instead just displayes a white screen on every step:

![image](https://user-images.githubusercontent.com/57028336/227076930-aa6d9240-8b34-4242-ba43-500e05ee7b4d.png)

if backwards compatibility is not intended/guaranteed (which is understandable), i think a warning should be displayed when viewing traces generated with a different playwright version. this is a common occurrence when using `trace.playwright.dev` which is often on a different version to the playwright version my project is currently on

**Comment by beegotsy** [comment]
I am using `playwright-java@1.21.0` and trace files are compatible with `playwright/test@1.29.2` but not compatible with the current trace viewer in https://trace.playwright.dev/.
As a work-around, if you have node in the server, you can use the show-trace command of playwright:
```
npm install -g @playwright/test@1.29.2
npx playwright show-trace
```
more informations here: https://playwright.dev/docs/trace-viewer

**Comment by xiang-deng** [comment]
Hi, actually the demo linked provided in the [doc](https://playwright.dev/docs/trace-viewer) has this issue. https://trace.playwright.dev/?trace=https://demo.playwright.dev/reports/todomvc/data/cb0fa77ebd9487a5c899f3ae65a7ffdbac681182.zip

**Comment by pavelfeldman** [comment]
Should be good now on trace.playwright.dev. npx playwright show-trace fix will be available in 1.32.1 early next week.
